src/translations/components/TranslationsProductTypesPage/TranslationsProductTypesPage.tsx

```diff
@@ -1,22 +1,39 @@
 import AppHeader from "@saleor/components/AppHeader";
+import CardSpacer from "@saleor/components/CardSpacer";
 import Container from "@saleor/components/Container";
 import LanguageSwitch from "@saleor/components/LanguageSwitch";
 import PageHeader from "@saleor/components/PageHeader";
-import { AttributeTranslationFragment } from "@saleor/fragments/types/AttributeTranslationFragment";
+import { AttributeTranslationDetailsFragment } from "@saleor/fragments/types/AttributeTranslationDetailsFragment";
 import { commonMessages, sectionNames } from "@saleor/intl";
 import { TranslationsEntitiesPageProps } from "@saleor/translations/types";
+import { ListSettings } from "@saleor/types";
 import React from "react";
-import { useIntl } from "react-intl";
+import { defineMessages, useIntl } from "react-intl";
 
 import {
   AttributeInputTypeEnum,
   LanguageCodeEnum
 } from "../../../types/globalTypes";
 import TranslationFields, { TranslationField } from "../TranslationFields";
 
+export const messages = defineMessages({
+  values: {
+    defaultMessage: "Values",
+    description: "section name"
+  }
+});
+
 export interface TranslationsProductTypesPageProps
   extends TranslationsEntitiesPageProps {
-  data: AttributeTranslationFragment;
+  data: AttributeTranslationDetailsFragment;
+  settings?: ListSettings;
+  onUpdateListSettings?: (key: keyof ListSettings, value: any) => void;
```

Comment:
is there no way to narrow down the value types?

Reply:
there is 👍 
https://github.com/mirumee/saleor-dashboard/pull/1165